bool get_status
bool is_cancel
bool is_charging
---
string status_is
